test/functions_test.dart

```diff
@@ -317,47 +318,49 @@ void main() {
     test('returns separate sets for each strongly connected component', () {
       // https://en.wikipedia.org/wiki/Strongly_connected_component#/media/File:Scc.png
       expect(
-          stronglyConnectedComponents({
-            'a': ['b'],
-            'b': ['c', 'e', 'f'],
-            'c': ['d', 'g'],
-            'd': ['c', 'h'],
-            'e': ['a', 'f'],
-            'f': ['g'],
-            'g': ['f'],
-            'h': ['g', 'd']
-          }),
-          equals([
-            // This is expected to return *a* topological ordering, but this isn't
-            // the only valid one. If the function implementation changes in the
-            // future, this test may need to be updated.
-            {'a', 'b', 'e'},
-            {'c', 'd', 'h'},
-            {'f', 'g'},
-          ]));
+        stronglyConnectedComponents({
```

Comment:
Are we missing some config to check formatting on PRs? I'm surprised this got through.

Reply:
I added a trailing comma to get things below 80 cols.